canonical-websites/www.ubuntu.com
# Open stack page updates 4206

## Done

Updated Text corrections according to the copy doc.
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Review [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit) and make sure that text matches copydoc


## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4206
